src/kakarot/execution_context.cairo

```diff
@@ -339,10 +339,7 @@ namespace ExecutionContext {
         let updated_event_len = event.keys_len + 1;
 
         emit_event(
-            keys_len=updated_event_len,
-            keys=event_keys,
-            data_len=event.data_len,
-            data=event.data,
+            keys_len=updated_event_len, keys=event_keys, data_len=event.data_len, data=event.data
```

Comment:
I ran `make format` and got this formatting, so I assume this file was not formatted in some previous PR! 

Hence pushing this formatting change as well!

Reply:
it's because you removed the last trailing comma, which enfores the args to be separated one in each line